src/sidebar/store/modules/selection.js

```diff
@@ -73,6 +73,13 @@ function initialState(settings) {
      * @type {SortKey}
      */
     sortKey: TAB_SORTKEY_DEFAULT.annotation,
+
+    /**
+     * ID or tag of an annotation that should be given keyboard focus.
+     *
+     * @type {string|null}
+     */
+    focusRequest: null,
```

Comment:
While our various dev affordances (i.e. editor integrations) will quickly expose to us that `focusRequest` is typed `string|null`, its name alone doesn't scream "identifier for an annotation" to me.